lib/src/line_sdk.dart

```diff
@@ -135,9 +133,9 @@ class LineSDK {
   ///
   /// {@macro error_handling}
   Future<UserProfile> getProfile() async {
-    String result = await channel.invokeMethod('getProfile');
-    if (result == null) return null;
-    return UserProfile._(json.decode(result));
+    return await channel
+        .invokeMethod('getProfile')
+        .then((value) => UserProfile._(json.decode(value ?? {})));
```

Comment:
maybe extract `json.decode(value ?? {})` to following function?
```
  dynamic decodeJson(String? source) {
    if (source != null) {
      return json.decode(source);
    } else {
      return {};
    }
  }
```

Reply:
Done!